lib/zipkin-tracer/tracer_factory.rb

```diff
@@ -6,7 +6,11 @@ def tracer(config)
       tracer = case adapter
         when :json
           require 'zipkin-tracer/zipkin_http_sender'
-          options = { json_api_host: config.json_api_host, logger: config.logger }
+          options = {
+            json_api_host: config.json_api_host,
+            async: config.async,
```

Comment:
can you puts async first so the order is alphabetical (like the SQS sender)?

Reply:
ok